example_batch_test.go

```diff
@@ -3,8 +3,9 @@ package gocql_test
 import (
 	"context"
 	"fmt"
-	"github.com/gocql/gocql"
```

Comment:
You can replace it in-place rather than as a separate import line. Same for other files.

Reply:
I did only change in-place but when I save the file VS-CODE itself does this change.